src/utils/GraphhopperUtils.js

```diff
@@ -285,7 +271,7 @@ async function fetchRoutes(end: string, start: string, departureTimeDateNow: str
 
   if (busRouteBufferedFirstRequest && busRouteBufferedFirstRequest.statusCode < 300) {
     const bufferedBusRoutes = JSON.parse(busRouteBufferedFirstRequest.body).paths;
-    busRoutes = getValidBusRoutes(busRoutes, bufferedBusRoutes).concat(busRoutes);
+    routes = getValidBusRoutes(routes, bufferedBusRoutes).concat(routes);
```

Comment:
also not in this pr but is there a reason why this can't be `routes = routes.concat(getValidBusRoutes(routes, bufferedBusRoutes)`?

Reply:
the buffered requests return routes before the start time, so I'm guessing we want those routes first